src/sql/run.py

```diff
@@ -154,6 +154,23 @@ def _repr_html_(self):
         else:
             return None
 
+    def _repr_text(self):
+        if self.pretty:
+            self.pretty.add_rows(self)
+            result = str(self.pretty)
+            if self.truncated:
+                result += (
+                    "\nTruncated to display limit of {}\n"
```

Comment:
This message repeats itself (`_repr_html_`). Let's export this message to a function `_get_truncated_message(html=False) -> str`, so we can control the truncated message in one place.

This function should return the truncated message as a string (for the terminal). If `html=True` this function should return the html we use in `_repr_html_`.

Then we can modify the code as follows:

```python
def _repr_html_(self):
    ...
    if self.truncated:
        HTML = _get_truncated_message(html=True)
```

```python
def __repr__(self) -> str:
    ...
    if self.truncated:
        message = _get_truncated_message()
        result = f"{result} {message}"
```

Reply:
Add `_get_truncated_message(html=False) -> str` function and update it it `_repr_html_` and `__repr__`